lib/matplotlib/backends/backend_agg.py

```diff
@@ -468,8 +467,7 @@ def _print_pil(self, filename_or_obj, fmt, pil_kwargs, metadata=None):
             filename_or_obj, self.buffer_rgba(), format=fmt, origin="upper",
             dpi=self.figure.dpi, metadata=metadata, pil_kwargs=pil_kwargs)
 
-    @_api.delete_parameter("3.5", "args")
-    def print_png(self, filename_or_obj, *args,
+    def print_png(self, filename_or_obj,
```

Comment:
Once you expire a delete_parameter, all subsequent parameters also become kwonly (as it was impossible to pass them positionally during the deprecation period without triggering the deprecation). Ditto for all other insurances below.

Reply:
I was thinking about that. Makes sense!